simulator/config/two-sigma-settings.edn

```diff
@@ -0,0 +1,65 @@
+{:sim-db-uri "datomic:free://localhost:4334/cook-sim"
+ :cook-db-uri "datomic:free://localhost:4334/cook-jobs"
+ :cook-api-uri "http://localhost:12321"
+ :process-count 10
+ :sim-model {:label "Two Sigma's cluster usage"
+             :duration-seconds 3600
+             :researcher-profiles
+             [{:description "Simulation runners"
+               :usernames ["simmer1" "simmer2" "simmer3" "simmer4" "simmer5"
+                           "simmer6" "simmer7" "simmer8" "simmer9" "simmer10"]
+               :seconds-between-jobs {:mean 7.2
+                                      :std-dev 0.72
+                                      :floor 1
+                                      :ceiling 120}
+               :job-duration {:mean 1800
+                              :std-dev 300
+                              :floor 1
+                              :ceiling 5400}
+               :job-memory {:mean 40960
+                            :std-dev 10240
+                            :floor 1024
+                            :ceiling 512000}
+               :job-cpu {:mean 3.0
+                         :std-dev 1.0
+                         :floor 1.0
+                         :ceiling 8.0}}
+
+              {:description "Fcat users"
```

Comment:
Well - this file is actually supposed to proportionally mimic Two Sigma's specific use case.  I wonder, is it even a good idea to have such a file in this repo at all?

I think it will be difficult to concisely describe the profiles we are talking about in a way that would be suitable for a label on a graph... for example, "Spark users" are unusual in these sims because the job-duration is always exactly 7200 seconds.  "Long running, consistent runtime, high memory, consistent CPU" wouldn't be good label on a graph...  this string is really supposed to describe WHY the collection of attributes is the way it is, rather than just summarize the attributes.

The problem would only get worse if there were a lot more profiles, or the behavior of each profile was modelled in a more sophisticated way.